Equation.java

```diff
@@ -0,0 +1,56 @@
+import static java.lang.Math.*;
+/**
+ * solve the quadratic equation
+ * @author Babak Alexey
+ */
+public class Equation {
+
+   /** 
+    * check possibility of the "String -> Double" conversion
+    * @param string  the string to check,
+    * if string has only double variable  - return it
+    */
+   public static double convertString(String string) {	
+      double coeff;
+      try {
+         coeff = Double.parseDouble(string);
+         return coeff ;
+      } catch (Exception e) {
+         System.out.println("Try to input every number in decimal type");

```

Comment:
Why method that must just convert stop overall application? It's quite strange and not documented